bundles/net.tourbook/src/net/tourbook/ui/TreeColumnFactory.java

```diff
@@ -199,6 +202,26 @@ public TreeColumnDefinition createColumn(final ColumnManager columnManager,
          }
       };
 
+      AVG_ALTITUDE_CHANGE = new TreeColumnFactory() {
+         @Override
+         public TreeColumnDefinition createColumn(final ColumnManager columnManager,
+                                                  final PixelConverter pixelConverter) {
+
+            final TreeColumnDefinition colDef = new TreeColumnDefinition(columnManager, "AVG_ALTITUDE_CHANGE", SWT.TRAIL); //$NON-NLS-1$
+            final String unitLabel = UI.SYMBOL_AVERAGE + UI.SPACE + UI.UNIT_LABEL_ALTITUDE + "/" + UI.UNIT_LABEL_DISTANCE; //$NON-NLS-1$
+
+            colDef.setColumnCategory(Messages.ColumnFactory_Category_Altitude);
+            colDef.setColumnLabel(Messages.ColumnFactory_avg_altitude_change_label);
+            colDef.setColumnHeaderText(unitLabel);
+            colDef.setColumnUnit(unitLabel);
+            colDef.setColumnHeaderToolTipText(Messages.ColumnFactory_avg_altitude_change_tooltip);
```

Comment:
My suggestion `ColumnFactory_Altitude_AvgChange_Label/Tooltip` similar to 
`ColumnFactory_Temperature_ ... _Label/_Tooltip`

Reply:
Done (see PR)